GVRf/Framework/src/org/gearvrf/GVRCameraRig.java

```diff
@@ -156,6 +159,8 @@ public float getFloat(String key) {
      *            The {@code float} value to map.
      */
     public void setFloat(String key, float value) {
+        checkKeyNotNullOrEmpty(key);
+        checkNotNaN(value);
```

Comment:
To GL, infinity is not a valid value. But the reason why I just check NaN is because NaN can be used as an illegal state identifier when we try to throw Java exception. I mean I am trying to balance the overhead of this class and how many conditions we should check with this class. Maybe we can discuss about it tomorrow.